evm/rlp/transactions.py

```diff
@@ -161,6 +164,10 @@ class BaseShardingTransaction(rlp.Serializable):
         ('code', binary),
     ]
 
+    def __init__(self, chain_id, shard_id, to, data, gas, gas_price, access_list, code):
+        super().__init__(chain_id, shard_id, to, data, gas, gas_price, access_list, code)
```

Comment:
You could `*args, **kwargs` both this call and the `__init__` signature here since you aren't using any of these variables in the local context.